LanguageFeatures/Extension-types/static_analysis_extension_types_A17_t03.dart

```diff
@@ -2,31 +2,28 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion Let DV be an extension type declaration named Name with type
-/// parameters X1 extends B1, .. Xs extends Bs. Assume that the representation
-/// declaration of DV is (R id)
+/// @assertion An extension type `V` is a proper subtype of `Object?`. It is
+/// potentially non-nullable, unless it implements `Object` or a subtype thereof
 ///
-/// We then say that the declared representation type of Name is R, and the
-/// instantiated representation type corresponding to Name<T1,.. Ts> is
-/// [T1/X1, .. Ts/Xs]R.
-/// ...
-/// Let V be an extension type of the form Name<T1, .. Ts>, and let R be the
-/// corresponding instantiated representation type. If R is non-nullable then V
-/// is a proper subtype of Object, and V is non-nullable. Otherwise, V is a
-/// proper subtype of Object?, and V is potentially nullable.
-///
-/// @description Checks that if an instantiated representation type `R` is
-/// not non-nullable then it is a proper subtype of `Object?` (can be assigned
-/// to `Object?`)
+/// @description Checks that if an extension type doesn't implement subtype of
```

Comment:
Perhaps:
```suggestion
/// @description Checks that even if an extension type doesn't implement a subtype of
```
The word 'even' is there to indicate that the following isn't implied: "If an extension type `E` does implement a subtype of `Object` then `E` isn't a subtype of `Object?`".

Reply:
There are a couple of other occurrences of 'implement subtype'.